Solutions/Commvault Security IQ/Package/mainTemplate.json

```diff
@@ -0,0 +1,874 @@
+              "name": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',concat('AnalyticsRule-', last(split(variables('analyticRuleId3'),'/'))))]",
+              "properties": {
+                "description": "Commvault Security IQ Analytics Rule 3",
+                "parentId": "[variables('analyticRuleId3')]",
+                "contentId": "[variables('_analyticRulecontentId3')]",
+                "kind": "AnalyticsRule",
+                "version": "[variables('analyticRuleVersion3')]",
+                "source": {
+                  "kind": "Solution",
+                  "name": "Commvault Security IQ",
+                  "sourceId": "[variables('_solutionId')]"
+                },
+                "author": {
+                  "name": "svc.cv-securityiq@commvault.com"
+                },
+                "support": {
+                  "tier": "Partner",
+                  "name": "Commvault",
+                  "email": "support@commvault.com",
+                  "link": "https://www.commvault.com/support"
+                }
+              }
+            }
+          ]
+        },
+        "packageKind": "Solution",
+        "packageVersion": "[variables('_solutionVersion')]",
+        "packageName": "[variables('_solutionName')]",
+        "packageId": "[variables('_solutionId')]",
+        "contentSchemaVersion": "3.0.0",
+        "contentId": "[variables('_analyticRulecontentId3')]",
+        "contentKind": "AnalyticsRule",
+        "displayName": "User Alert",
+        "contentProductId": "[variables('_analyticRulecontentProductId3')]",
+        "id": "[variables('_analyticRulecontentProductId3')]",
+        "version": "[variables('analyticRuleVersion3')]"
+      }
+    },
+    {
+      "type": "Microsoft.OperationalInsights/workspaces/providers/contentTemplates",
+      "apiVersion": "2023-04-01-preview",
+      "name": "[variables('playbookTemplateSpecName1')]",
+      "location": "[parameters('workspace-location')]",
+      "dependsOn": [
+        "[extensionResourceId(resourceId('Microsoft.OperationalInsights/workspaces', parameters('workspace')), 'Microsoft.SecurityInsights/contentPackages', variables('_solutionId'))]"
+      ],
+      "properties": {
+        "description": "Commvault-Logic-App10 Playbook with template version 3.0.0",
+        "mainTemplate": {
+          "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+          "contentVersion": "[variables('playbookVersion1')]",
+          "parameters": {
+            "PlaybookName": {
+              "defaultValue": "Commvault-Logic-App10",
+              "type": "string"
+            },
+            "Automation-Account": {
+              "defaultValue": "Commvault-Automation-Account",
+              "type": "string"
+            },
+            "keyvaultName": {
+              "type": "string"
+            }
+          },
+          "variables": {
+            "location": "[[resourceGroup().location]",
```

Comment:
Please change the "location" to this value `"[parameters('workspace-location')]"`

Reply:
I've made the required changes and pushed them to the PR.

Reply:
The SolutionValidations is failing, can you please take a look?